packages/babel-helper-validator-option/src/validator.ts

```diff
@@ -46,13 +45,13 @@ export class OptionValidator {
     return value;
   }
 
-  validateStringOption(
+  validateStringOption<T>(
     name: string,
     value?: string,
```

Comment:
Actually, `value` could be anything here I think (otherwise we wouldn't need this helper).

Reply:
Ideally `value` should be of type that extends `string`. So if we accidentally pass `number` value to `validateStringOption`, the type checker should throw.

Reply:
Couldn't it be anything, since it's whatever the user gave us as input? We cannot statically know the type of plugin options, so an user could pass a number to a plugin, the plugin will pass a number to this helper, and then this helper will throw.

Reply:
We also offer our type interface to programmatic users:
```ts
{ a: any, b: any }
```
is technically correct (which is the type user inputs) but not much of value for programmatic usage (API caller) and/or schema check. The `validator` here is a runtime enforcement of type checking -- so the user input does match our assumption of types, so it makes sense that if the type checking is good, it is also sound in runtime.